src/components/Signup.tsx

```diff
@@ -0,0 +1,41 @@
+import { auth } from 'config/firebase';
+
+import React, { useRef } from 'react';
+
+export default function Signup() {
```

Comment:
export default isn't good for components because it cause tree shaking. It is better to use only export

Reply:
When I remove the default the application is not working.